PeerioTechnologies/peerio-mobile
# feat: Add file preview to files upload

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/1769/mobile-upload-from-moments
#### Testing instructions  
File preview is now a stand-alone component, reused in (1) share in channel and (2) upload in files.
Test feature: upload a photo from files tab, see preview of image.
Test for regression: share a file in a room, see preview of image.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
